uhi/typing/plottable.py

```diff
@@ -117,11 +117,12 @@ def variances(self) -> Optional[ArrayLike]:
         kind=="MEAN".
         """
 
-    def counts(self) -> Optional[ArrayLike]:
+    def effective_counts(self) -> Optional[ArrayLike]:
```

Comment:
This is a bit a matter of taste, but weighted histograms are a generalization of normal histograms. I would design the interface so that it has obvious names for normal histograms without weights.

Reply:
I thought you said brevity was not important for a protocol? I think that it's going to be very confusing when users see `.counts()` but it is neither the counts nor the sum of weighed counts. No user is going to go look up our protocol, they will only see the library. Plus, every time you referred to this you called it "effective counts" since that is the actual description of what this means.

Reply:
I understand that point of view, but "count/counts" has such a strong sense of being a discrete number that "bin 12 counted 17.554 entries" sounds too bizarre. I think that's why this quantity was named "effective entries" in HBOOK, back when Fortran names were limited to 6 characters.

Reply:
I do see the generalization argument, so I would consider myself a mild supporter of `effective_counts`. @HDembinski and @jpivarski please come to an agreement, and I'll go with whatever you decide.

Sorry to bring this up after the meeting, but we did change the meaning of counts (for the better) without discussing the name.

Reply:
There are some parts of the protocol that nornal users won't touch, but .values, .variances and .counts will be touched frequently. You will type that a lot, hemce short easy to remember names are better here. I made my arguments yesterday always under the implicit understanding that the names do not change.

Reply:
What .values, .counts etc means is specified by the protocol and is going to documented in docstrings. There no need here to use an unwieldly but very explicit name. Also .counts is really just the number of entries in that cell for an unweighted profile.

Reply:
Again, I was 100 % happy with the compromise reached yesterday, I want nothing to change.

Reply:
Let's revert `effective_counts` to `counts`.

Reply:
Okay, will do.

Reply:
```suggestion
    def counts(self) -> Optional[ArrayLike]:
```